globus_cli/commands/login.py

```diff
@@ -13,6 +13,23 @@
     internal_auth_client, write_option)
 
 
+_LOGIN_EPILOG = ("""\
+
+SUCCESS
+You have logged in to the Globus CLI as {}
+
+A set of valid tokens for accessing Globus have been saved for you in
+~/.globus.cfg and are valid indefinitely unless revoked.
```

Comment:
Personally, I'd not mention anything about it. From the perspective of the user, why do they need/want to know all the details of what's going on under the hood?